src/lib/lineage-constraints.js

```diff
@@ -43,9 +43,9 @@ Ensure we are not creating a circular hierarchy
 */
 const getHierarchyViolations = (mapTypeToAllowedParents, contactDoc, parentDoc) => {
   const { type: contactType } = contactDoc;
```

Comment:
Shouldn't we do the same here? Trying to move a "configurable type" contact ends up with 
```
Error: Hierarchy Constraints: cannot move contact with unknown type 'contact'
```

Reply:
That's embarrassing